apis/logging/v1/clusterlogging_types.go

```diff
@@ -178,34 +173,67 @@ type ElasticsearchSpec struct {
 
 // This is the struct that will contain information pertinent to Log and event collection
 type CollectionSpec struct {
-	// Specification of Log Collection for the cluster
-	Logs LogCollectionSpec `json:"logs,omitempty"`
```

Comment:
Same here.  This will need to be marked deprecated for for backwards compatibility reasons